### DISCUSSION TITLE
Repository Format Page Link error

### DISCUSSION THREAD

**Discussion by isaacriehm** [community/community#58420] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.117.0 (101615344)
Device: iPhone15,2 (iOS 16.5)
</details>

When a page link in the format of a repository (https://.../username/repo) is clicked on iOS Safari, the GitHub app takes over the link and then displays a 404 because it tries to search for the repository.

Example:

https://github.com/github-copilot/chat_waitlist_signup

![image](https://github.com/community/community/assets/32783392/6c7817e7-c10f-4d67-a331-077fe54e855c)
![image](https://github.com/community/community/assets/32783392/cc84f003-c2fc-4321-ad1d-fe0750256b1a)

**Comment by iskae** [comment, marked as answer]
Hi @IsaacTheDev 👋 

Thanks a bunch for reporting this ❤️ Unfortunately this is not a straightforward fix, but I have created an internal tracking issue for this and hopefully we can come up with something as soon as possible 🙇 

Please stay tuned for the future versions for a potential fix 🙇 

Cheers!
